3_core_core_analysis/1_core_core_relationships_across_strains.ipynb

```diff
@@ -0,0 +1,1758 @@
5otm9rKZ9ZjZ/efJc5eZbZ3A+HzNzOLh+LSa2QtmdvmgYa4Nx+fPhnj9d81sn5klzOyuEX7P0+F7jOpuy5q2xjZtmVmBmf00XIg4M7t2mN9RYGZ7zrOAGHEBMorxuMvM+sPxOGtmO83slkHDLA6nmW8P8Xozs3vM7ICZxcyswcz+1swKk4a5zsyeNbM2M6sfbTZt0oaGmhFHmDn/3jlXBswDTgL3D3r+TqAF+Oyg9ysAngIWApcDFcA9wN+Z2R8nDXocuBe4b8wjMj4/CcdnBrAV+JmZJd8JZcjxCb0G/Adgx3BvbmafBvJTFzezTMW0FdoK/AEw0res3QOcOk/kVHgxHI9K4HvAg2Y2Pen5zwJngE8kF1non4AvhMOUAzcDNwAPJg3TSTB/3DOmVM65Ef8D6oE/BV4H2oCfAEXhc9cCR5OG3QC8CrQDD4XD3ps8LPAnBH/IRuDupNfenzTsG8BHkp7LB04DFw2T8aPATuAs8BZwU/h4LfALggnkIPBHSa/5GvBT4IHwdZ8f6rEhftc7OcOfPwx0JP28EEgAtwF9wOyk5z4XjnvpoPf8BNABTBv0+L3A/SP8bVYB3UB/+PrW8PEK4PsEE/Zh4KtAZJj3+BrwQNLPawAH1IQ/l4Z/z08CvcCmYd5nK3DXEI9XAPuBy8L3jWraSv20Neh9jgLXDvH4YmAPQYEcHea1pUAs/D0d4X+1QCHwdYKF8fHw34XDvMddwNZB7+kGph3Aws/y3wMngNuThl1OMD1fMug95xPcTOL6QY/fCNSfr8cG/hvtGt7HgZvCD2xdOELnCNdefh7+0aqAHwG3DhpsNsEMMJdg5v/moNYf8H2CJdWADwGNzrlXh/i9l4TD30OwNLmaYEYC+DHBH78WuB34GzO7PunlHyWYCCuBH47w2JDMrAz4NMGMOOCzwMvOuYcJJq5PJz33fuBx997bGj0MFBGs9Y2ac24P8O8Il6bOucrwqW8QfM5LgGvCTHcP/S7njE8hwd/2iHPudPjwxwgm+oeAXxGsYYzF3wDfZvi1Dk1bQxjHtDUa3wD+gqDQhhROmzcDx8Npqsw5dxz4CsFC60JgPXAJwYJ0ROGa7OcJpqED4cNXEqzB/phgrS15mrqBoIy3Dcp1BPgdwTw0bqMtvH9yzh13zrUAjxKM9GCXAdFw2Lhz7mfAtkHDxIH/Gj7/S4IPYaj9NA8AHzKzaeHPnwF+MEy2zwH3OeeedM4lnHPHnHN7zWw+cAXwZ865bufcTuBfOHdT4EXn3CPh62IjPDbYn5pZK8GSvYxzZ9LPAlvCf28Z9PtqCNY+zuGc6yNYy6gZ5veNmpnlEayN/blzrt05Vw/8T4LPcDgfD8fnCLCRc8vkToJN3n6C8fmkmY1q89TMNhH8Db4xwmCats413mlrRGZ2K5DnnPv5aF8zyKcJPt+TzrlTwH9h5GnqsnA8moBPAbc659rC5+4kWPCfIRiPm8xsZvjckPNIqJEJziOjLbzkpXMXwR9isFrgmAvXM0NHBg3THM7cI75XuET5LXCbmVUSLHGGWyLOJ1g9HipPi3OuPemxwwRrAMPlG+6xwf6Hc67SOTfbOfd7zrm3AMzsCoI1lR+Hw20B6sxsYCY+DcwZ/GbhUrAmfH6iagg20w4nPTZ4vAd7MByfmc65651zr4S55gPX8e5n/28Ea6IfPl8IC+6L9y3gy4P+5oNp2jrXeKetYZlZKfD3wH8cxe8fTi3vnaZqRxj+d+F41DjnLnPOPRVmKQbuIPzMnXMvAg3A5vB1Q84joTlMcB5J5UGLRmDuoJ3d8yfwfv+HYNPjDoIl47FhhjsCLB3i8eNAlZmVJz22AEh+n6FuBjiRGwTeSbB/YqeZNQEvJT0OwQGLm8MJMNltBPsnfjeO3zk472mCtZ2FSY8NHu/R+gzBNPJoOD6HCApvNJu104BNwE/C124PHz9qZleNMYemrfNPWyNZDiwCng9f+zNgjgVnCiwaZc7jvHeaOj6q5Oe6lWDa+Fb4+5sIFhQD4/EMMD/cnfCOcOF7GfD0OH7nO1JZeC8S7Gz8oplFzeyjBNv54/UIwY7qLxPsRxnO94C7zewGM4uY2VwzuyDc5n8B+FszKzKzdQSbKA9MINOwzKyIYH/UFwg2ywb++xKwOVyL+wHBfp+HzGyRmeWb2QcJjkp9bWCVP/z8ioA8IC/MP9xRvRPAvHA/F+Gm54PAfzOzcjNbCPzxOMf7ToJNl+TxuY1gk7A6zFoQZjUgP8waITgIUZv0ug+F77mRd2fW0dK0df5pCzMrDIcFKAizGcGBmvlJr/s8wXRzIUOvdZ4Aqs2sIumxHwFfNbMZZlYD/CXjn6buA+qS8lwBrDezOufcfuA7wA/N7DIzyzOzNQT7uZ9KWlOMhOOaH/xoRQPzwEhSVnjOuV6CHdyfA1oJlqCPEay5jOf9YgQjuZhgiTTccNsIdsj/L4KZ7P/x7pLoUwRLtuMEO73/auADmwS/T7Az+PvOuaaB/wj+uFGCo3s9BEeVjhDM9GeBfwS+4pz7h6T3+mr4Xv+Z4HOMMfwO4meA3UCTmQ2s7n+J4LD9IYKjp1sY4ykuZnYZwef4zeTxcc79gmD/0qfCQX8d5nsf8N3w31e7QPLnMHAqxIlwWhk1TVvnn7bC4faFw80lOMAUAxY65/oGva4FSIQ/v+dLj5xzewkK7pAF52bWEpwx8DLBEfVdBKchjelkbjObS3BQ4uuDpqlXgCd4dy3viwT7RB8g2Bf7BPAcwcJ2wNXh+P2SYG0zRjAtjpzh3N0iqWVmLwHfcc796zhf/5fACufcH5x3YMkpmrZkPFJ64rGZXWNms8PNjjsJTjN4YpzvVUWwRP9uKjNKZtK0JamQ6istVhKced9KcBLo7c654Q4xD8vM/ohgs+9x59xvUhtRMpSmLZkwfWuZiOQMXUsrIjlDhSciOUOFJyI5Q4UnIjlDhSciOUOFJyI54/8DyUjK//pujVkAAAAASUVORK5CYII=\n",
```

Comment:
I understand the point of the figure, that there is a very high degree of overlap, but this venn diagram is...difficult to understand as a venn diagram haha